ocpp/charge_point.py

```diff
@@ -170,20 +170,22 @@ def _raise_key_error(action, version):
     from ocpp.v201.enums import Action as v201_Action
 
     if version == "1.6":
-        if hasattr(v16_Action, action):
+        try:
```

Comment:
Any particular reason why we are changing this piece of code? I think the previous version will still work.
Will the change bring any additional functionality?

Reply:
@drc38, can we add a test in `test_charge_point.py` for the `_raise_key_error` function?

Reply:
> @drc38, can we add a test in `test_charge_point.py` for the `_raise_key_error` function?

This is already tested for, see https://github.com/mobilityhouse/ocpp/commit/f159cf7b664339a73c6733c35f43a9c38b558561

Reply:
Ahh. Missed that. Looks good then.